### DISCUSSION TITLE
Crash reporting in code-server?

### DISCUSSION THREAD

**Discussion by gauthamp10** [coder/code-server#2338] [Q&A] [answered]
I just went through the FAQ, guides and on VSCode documentation but code-server doesn't seems to have an option to disable crash reporting (inside vscode) other than disabling basic telemetry.

Furthermore, code-server vscode doesn't seem to have **Preferences: Configure Runtime Argument** in command palette to open up an argv.json to do this **"enable-crash-reporter": false** as mentioned here: [vscode/telemetry](https://code.visualstudio.com/docs/getstarted/telemetry)

Can anyone shed some light? 
Do we have a code-server built on top of vscodium as of now?

**Comment by code-asher** [comment, marked as answer]
I don't think we can build on vscodium; from what I understand vscodium
is just some scripts that build VS Code.

I don't think we could use those scripts either since we need to have a
special build process for web but it couldn't hurt to take a look and
see if there's anything we could make use of.

I took a look and the crash reporter code is specific to Electron so it
doesn't run in code-server.

**Reply by gauthamp10** [reply]
OK...thanks.
